Classes/Controller/OaiPmhController.php

```diff
@@ -107,8 +107,8 @@ public function initializeAction()
      */
     protected $formats = [
         'oai_dc' => [
-            'schema' => 'http://www.openarchives.org/OAI/2.0/oai_dc.xsd',
-            'namespace' => 'http://www.openarchives.org/OAI/2.0/oai_dc/',
+            'schema' => 'https://www.openarchives.org/OAI/2.0/oai_dc.xsd',
+            'namespace' => 'https://www.openarchives.org/OAI/2.0/oai_dc/',
```

Comment:
This is a XML namespace and should not be changed!
```suggestion
            'namespace' => 'http://www.openarchives.org/OAI/2.0/oai_dc/',
```